api/views/phones.py

```diff
@@ -640,6 +643,168 @@ def sms_status(request):
     return response.Response(status=200)
 
 
+call_body = openapi.Schema(
+    type=openapi.TYPE_OBJECT,
+    required=["to"],
+    properties={"to": openapi.Schema(type=openapi.TYPE_STRING)},
+)
+
+
+@decorators.permission_classes([permissions.IsAuthenticated, HasPhoneService])
+@swagger_auto_schema(method="post", request_body=call_body)
+@decorators.api_view(["POST"])
+def outbound_call(request):
+    """
+    Make a call from the authenticated user's relay number.
+
+    """
+    if not flag_is_active(request, "outbound_phone"):
+        return response.Response({"outbound_phone": False}, status=404)
+    to = request.data.get("to", None)
```

Comment:
**Blocking** TODO: We need to validate this input is a full, valid number. 

We may also need to write a special catch if the number is a public safety answering points (911, etc)

Reply:
I'm not sure we should allow 911 numbers. Twilio wants you to register an address for those, and the default may be to call the 911 call center associated with the relay number, which would probably be the wrong thing, unless you were intending to make a fraudulent call.

@groovecoder do you have an opinion on 911 or other N11 codes? https://en.wikipedia.org/wiki/N11_code

Reply:
Right, the "special catch" for a PSAP should be that we *don't allow* those calls.

Reply:
OK. Remove all N11 numbers, or just 911?

Reply:
Outside of GitHub, @maxxcrawford and @groovecoder verified that then mean all the N11 service codes should be invalid. All of these services route to a location-specific provider of a service, and don't make sense when the "location" of a mask is different from the user's location, unless the user is trying to hide their location, which seems much more likely to be fraud.

I've remove the code that allows N11 service numbers, and kept two tests that check if a service code is allowed (with mocking of the Twilio service).